k9mail/src/main/java/com/fsck/k9/notification/AuthenticationErrorNotifications.java

```diff
@@ -35,7 +35,7 @@ public void showAuthenticationErrorNotification(Account account, boolean incomin
 
         NotificationCompat.Builder builder = controller.createNotificationBuilder()
                 .setSmallIcon(R.drawable.notification_icon_warning)
-                .setWhen(System.currentTimeMillis())
+                .setWhen(SystemClock.elapsedRealtime())
```

Comment:
Also wrong - this isn't a duration it's a date-time so it needs to use currentTimeMillis.